src/lib/mongoBackend/mongoGetSubscriptions.cpp

```diff
@@ -173,7 +176,7 @@ void mongoListSubscriptions
   std::string                    conds = std::string(CSUB_CONDITIONS) + "." + CSUB_CONDITIONS_TYPE;
   BSONObj                        q     = BSON(conds << "ONCHANGE");
 
-  if (!collectionQuery(getSubscribeContextCollectionName(tenant), q, &cursor, &err))
+  if (!collectionRangedQuery(getSubscribeContextCollectionName(tenant), q, limit, offset, &cursor, &count, &err))
```

Comment:
Given the way in which _id is auto-build, it acts as a creation date, so you get the same effect (note that, different from documents in the entities collection, which has a `creDate` field, registrations and subscriptions didn't have).